changes.d/5704.fix.md

```diff
@@ -0,0 +1,2 @@
+Fix bug where runahead limit was set to max active cycle points by
+the configuration converter.
```

Comment:
Maybe be more specific so that users don't have to figure out this is Cylc 7 -> 8 conversion (if they ever read change logs 🤣 ). E.g.
```suggestion
Fix off-by-one error in automatic upgrade of Cylc 7 "max active cycle points" to Cylc 8 "runahead limit".
```